app/grandchallenge/cases/widgets.py

```diff
@@ -110,12 +110,15 @@ class FlexibleImageField(MultiValueField):
     def __init__(  # noqa C901
         self,
         *args,
+        interface,
         user=None,
         initial=None,
         **kwargs,
     ):
         image_search_queryset = filter_by_permission(
-            queryset=Image.objects.all(),
+            queryset=Image.objects.filter(
+                dicom_image_set__isnull=not interface.is_dicom_image_kind

```

Comment:
Double negation, my poor head. I guess these odd queries are the result of our choice to make it a fundamental part of the existing `Image`.